commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderServiceImpl.java

```diff
@@ -855,6 +819,21 @@ else if (commerceAccount.isBusinessAccount()) {
 		}
 	}
 
+	private long[] _getCommerceAccountIds(long groupId) throws PortalException {
+		long[] commerceAccountIds = null;
+
+		if (!PortalPermissionUtil.contains(
+			getPermissionChecker(),
+			CommerceAccountActionKeys.MANAGE_ALL_ACCOUNTS)) {
+
+			commerceAccountIds =
```

Comment:
I think this logic is backwards -- if user does not have permission, set the accountIds!

Reply:
That's how I had it before @marco-leo had me change it during our call ;). I think he preferred it this way as it reduces a needless call in the case that the user does have MANAGE_ALL_ACCOUNTS perm.

Reply:
Oh I see -- alright! Then it'd be simpler to just return the appropriate value rather than setting it to a var and then returning the var, but that's not really worth sending a separate PR for -- just something to keep in mind for the future.